src/hooks/useFetch.jsx

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from "react";
```

Comment:
Is this your code? Are you able to explain it? It is called useFetch but it uses Axios. There is a lot of unfamiliar code here. If you understand/can explain it then great! Otherwise I would refactor to the way we have shown and is in the notes 👍